Document _origin. (#129)
Zilliqa/scilla-docs: docs/source/scilla-in-depth.rst

    - ``_sender : ByStr20`` : The account address that triggered this
      transition. If the transition was called by a contract account
      instead of a user account, then ``_sender`` is the address of
      the contract that called this transition. In a chain call, this is

>       always the immediate caller.

Reviewer: ```suggestion
      the contract that sent the message invoking the current transition.
```

>     - ``_origin : ByStr20`` : The account address that triggered this

Reviewer: ```suggestion
    - ``_origin : ByStr20`` : The account address that initiated the current
```

      transaction (which can possibly be a chain call). This is always

>       a non-contract address.

Reviewer: ```suggestion
      a user address, since contracts can never initiate transactions.
```
